.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
>Although I admit that right know I don't know precisely what I want to enforce and what I don't want to enforce.

I'm in the same boat. I want to enforce more, but I can't bring myself to eliminate all of the violations in the code base. I'll bump up the 

>deally, I'd like to be able to enforce some rules (such as "add proper comments to your functions") and disable others (many of those related purely to style). Something like this, although not that particular set of rules. 

That's what the .pylintrc file is for. I would like to eliminate any per-file configuration.